lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -541,6 +541,7 @@ def add_external_three_d_secure_auth_data(post, options = {})
       def setup_future_usage(post, options = {})
         post[:setup_future_usage] = options[:setup_future_usage] if %w(on_session off_session).include?(options[:setup_future_usage])
         post[:off_session] = options[:off_session] if off_session_request?(options)
+        post[:expand] = ['charges.data.balance_transaction']
```

Comment:
I would also recommend moving this out of here and directly into `create_intent` since this is no longer related to just offsite transactions.

Reply:
@almalee24 so when i try to pass it in the create_setup_intent method, I receive the error "This property cannot be expanded (charges)." Not sure if that is supported for that endpoint

Reply:
Okay cool, thanks for checking that!

Reply:
I noticed you removed the remote test but I think if you just add `assert purchase.params.dig('charges', 'data')[0]['balance_transaction']` to like `test_successful_purchase` it would be good enough!